entity-services/src/test/java/com/marklogic/entityservices/tests/TestInstanceConverterGenerator.java

```diff
@@ -221,10 +221,10 @@ public void testConversionModuleExtractions() throws TestEvalException, JsonProc
             String controlFilePath = "/test-instances/" + entityTypeTestFileName;
             Document controlDom = builder.parse(this.getClass().getResourceAsStream(controlFilePath));
 
-            // logger.debug("Control doc");
-            // debugOutput(controlDom);
-            // logger.debug("Actual doc wrapped");
-            // debugOutput(actualInstance);
+             logger.debug("Control doc");
+             debugOutput(controlDom);
+             logger.debug("Actual doc wrapped");
+             debugOutput(actualInstance);
```

Comment:
no, I need to comment that out again, good catch. tests are rather more verbose with these uncommented :)